src/vs/workbench/api/common/extHost.protocol.ts

```diff
@@ -1315,6 +1325,11 @@ export interface MainThreadShareShape extends IDisposable {
 	$unregisterShareProvider(handle: number): void;
 }
 
+export interface MainThreadMappedEditsShape extends IDisposable {
```

Comment:
Given this is about language servers / language smarts, I suggest moving these into `MainThreadLanguageFeaturesShape` / `ExtHostLanguageFeaturesShape` like all the other language smarts